python/lsst/meas/algorithms/readTextCatalogTask.py

```diff
@@ -54,6 +54,22 @@ class ReadTextCatalogConfig(pexConfig.Config):
         doc=("Format of files to read, from the astropy.table I/O list here:"
              "http://docs.astropy.org/en/stable/io/unified.html#built-in-table-readers-writers")
     )
+    fill_values = pexConfig.ListField(
+        dtype=str,
+        default=None,
+        optional=True,
+        doc=("A list giving [<match_string>, <fill_value>], which is used to mask"
+             " the given values in the input file. '0' is suggested for the fill value in order to prevent"
+             " changing the column datatype. The default behavior is to fill empty data with zeros. See "
+             "https://docs.astropy.org/en/stable/io/ascii/read.html#bad-or-missing-values for more details."
+             "Use `replace_missing_floats_with_nan` to change floats to NaN instead of <fill_value>.")
+    )
+    replace_missing_floats_with_nan = pexConfig.Field(
+        dtype=bool,
+        default=False,
+        doc="If True, replace missing data in float columns with NaN instead of zero. If `fill_values` is "
+            "set, this parameter with replace the floats masked by `fill_values`."
```

Comment:
"with replace"->"will replace"

I don't think I understand that final sentence. Is this implying `replace_missing_floats_with_nan` takes precedence over `fill_values` (which I think is what we want)?